providers/celery/src/airflow/providers/celery/executors/celery_executor_utils.py

```diff
@@ -120,8 +120,9 @@ def on_celery_import_modules(*args, **kwargs):
     """
     import jinja2.ext  # noqa: F401
 
-    import airflow.jobs.local_task_job_runner
-    import airflow.macros
+    with contextlib.suppress(ImportError):
```

Comment:
```suggestion
    if not AIRFLOW_V_3_0_PLUS:
```

better?

Reply:
I guess so, yeah?  I'll see if tests pass and if there's any other feedback